src/sentry/south_migrations/0293_auto__del_unique_release_project_id_version.py

```diff
@@ -0,0 +1,711 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Removing unique constraint on 'Release', fields ['project_id', 'version']
+        db.delete_unique(u'sentry_release', ['project_id', 'version'])
```

Comment:
fyi if i try to do the DROP INDEX CONCURRENTLY, this is what happens:
```
 - Migrating forwards to 0293_auto__del_unique_release_project_id_version.
 > sentry:0293_auto__del_unique_release_project_id_version
   = DROP INDEX CONCURRENTLY sentry_release_project_id_7309d47e8f37e7ff_uniq []
FATAL ERROR - The following SQL query failed: DROP INDEX CONCURRENTLY sentry_release_project_id_7309d47e8f37e7ff_uniq
The error was: InternalError('cannot drop index sentry_release_project_id_7309d47e8f37e7ff_uniq because constraint sentry_release_project_id_7309d47e8f37e7ff_uniq on table sentry_release requires it
HINT:  You can drop constraint sentry_release_project_id_7309d47e8f37e7ff_uniq on table sentry_release instead.\n',)
SQL: DROP INDEX CONCURRENTLY sentry_release_project_id_7309d47e8f37e7ff_uniq
```